external/anomaly/ote_anomalib/tools/sample.py

```diff
@@ -244,25 +300,49 @@ def parse_args() -> Namespace:
     parser = argparse.ArgumentParser(
         description="Sample showcasing how to run Anomaly Classification Task using OTE SDK"
     )
-    parser.add_argument("--model_template_path", default="./anomaly_classification/configs/padim/template.yaml")
+    parser.add_argument(
+        "--model_template_path",
+        default="./anomaly_classification/configs/padim/template.yaml",
+    )
     parser.add_argument("--dataset_path", default="./datasets/MVTec")
     parser.add_argument("--category", default="bottle")
+    parser.add_argument("--optimization", choices=("none", "pot", "nncf"), default="none")
     parser.add_argument("--seed", default=0)
     return parser.parse_args()
 
 
-def main() -> None:
+def main(category) -> None:
     """Run `sample.py` with given CLI arguments."""
     args = parse_args()
-    path = os.path.join(args.dataset_path, args.category)
+    path = os.path.join(args.dataset_path, category)
 
     task = OteAnomalyTask(dataset_path=path, seed=args.seed, model_template_path=args.model_template_path)
 
     task.train()
     task.export()
+
     task.optimize()
+
+    task.optimize_nncf()
+    task.export_nncf()
+
     task.clean_up()
 
+    with open(f"/home/adokucha/padim/{category}.txt", "w") as file1:
+        for k, v in task.results.items():
+            # Writing data to a file
+            print(f"{k}: {v}")
+            file1.write(f"{k}: {v}\n")
+
 
 if __name__ == "__main__":
-    main()
+    # main()
+
+    data_dir = "/mnt/hdd2/datasets/MVTec/"
```

Comment:
Left over path?

Reply:
My fault, uploaded script to collect metrics